app/features/IdentitySlice.ts

```diff
@@ -58,6 +71,15 @@ export async function loadIdentities(dispatch: Dispatch) {
     dispatch(updateIdentities(identities));
 }
 
+export async function removeIdentityAndInitialAccount(
+    dispatch: Dispatch,
+    identityId: number
+) {
+    await removeIdentityAndInitialAccountInDatabase(identityId);
+    await loadAccounts(dispatch);
+    return dispatch(removeIdentityInRedux(identityId));
```

Comment:
I know we do this other places than this, and that removing an item from a list is fairly simple, but I'm not a fan of the duplication happening when we first change something in the DB, and afterwards replicate the change in redux as well.

Performance-wise, I know this is probably the best way, but it would be way simpler if we could have a general model for how updates are done to the DB and redux that relies on a single source for data. When we work with a local DB anyway, I don't think the performance gain is worth the added complexity (again, this action isn't complex).

What are your thoughts on this?

Reply:
I think you are correct, the performance gain does not justify adding the potential of de-synchronization between the db and redux.